### PR TITLE
CAMEL-12383 - changes regarding SB 2.0 upgrade

### PR DESCRIPTION
**Pull request by onderson**
I guess there is no more remaining changes required considering SB 2 upgrade.

### CODE REVIEW

File: archetypes/camel-archetype-spring-boot/src/main/resources/archetype-resources/src/main/resources/application.properties [hunk 1]
```diff
@@ -35,7 +35,7 @@ timer.period = 2000
 management.endpoints.web.exposure.include=info,health,camelroutes
 
 # all access to actuator endpoints without security
-management.security.enabled = false
+#management.security.enabled = false #is no longer customizable with SB 2.0
```
Comment (davsclaus) [inline archetypes/camel-archetype-spring-boot/src/main/resources/archetype-resources/src/main/resources/application.properties]: Should we maybe just delete all of this as its not possible with SB2 like this way?

Comment (onderson) [reply archetypes/camel-archetype-spring-boot/src/main/resources/archetype-resources/src/main/resources/application.properties]: considering support of former versions i just kept them commented. but i guess by 2.22.x we will drop and not dually support SB 1.X. so you are right. i will remove them.

**Comment by davsclaus** [comment]
Thanks @onderson looks good, you can merge and resolve the JIRA ticket

**Comment by onderson** [comment]
Thanks @davsclaus

**Comment by davidkarlsen** [comment]
@davsclaus Would it be helpful to add https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.0-Migration-Guide for a while - in order to detect any badly named properties?

**Comment by davidkarlsen** [comment]
@davsclaus 

Have a look in https://github.com/davidkarlsen/camel/tree/feature/springBootPropertiesMigrator
I guess all spring-boot code depends on this artefact, and hence if adding the checker here, it will propagate over to all consumers of the spring-boot code, and then validate the properties names to be spring-boot 2.x conformant. 

We only want this temporarily (until the release) / to be added during builds for validation (is there a profile which can be reused for this?) - to sanitycheck during build - and not propagate it transitively in the end-result of camel libraries so that it hits end-consumers.

**Comment by davsclaus** [comment]
@davidkarlsen yeah sure would you log a JIRA ticket and send a PR. The JIRA is there for us to not forget about remove it before release.

**Comment by davidkarlsen** [comment]
Would it be possible to have it in a profile which is only used for ci-builds or the likes (are there any such profiles as of today)? Then it could be there for future and also need not be removed.

**Comment by davsclaus** [comment]
The dependency AFAIK only logs a report about migration options. It does not fail a build etc (or can it be configured).

**Comment by onderson** [comment]
same for me. that's why i did locally and fixed all properties by following the guides but never tried to integrate such profile into mvn options.

**Pull request merged**